test/dummy/config/database.yml

```diff
@@ -11,15 +11,22 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: storage/development.sqlite3
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: storage/test.sqlite3
 
+
+# SQLite3 write its data on the local filesystem, as such it requires
+# persistent disks. If you are deploying to a managed service, you should
+# make sure it provides disk persistence, as many don't.
+#
+# Similarly, if you deploy your application as a Docker container, you must
+# ensure the database is located in a persisted volume.
 production:
   <<: *default
-  database: db/production.sqlite3
+  # database: path/to/persistent/storage/production.sqlite3
```

Comment:
Do we need to switch from `db` to storage? We'll need to purge our existing dbs.

Reply:
I figure it's better to since this is the new default, otherwise we'll need to update this manually easy time we re-generate the dummy app.